single-sign-on/manage/index.md

```diff
@@ -18,6 +18,30 @@ To add a guest to your organization in Docker Hub if they aren’t verified thro
 2. Select **Add Member**, enter the email address, and select a team from the drop-down list.
 3. Select **Add** to confirm.
 
+## Audit your domain for users
+
+When your organization has configured SSO and registered a domain, you can audit your domain. Auditing your domain will identify uncaptured users that have an email associated with your domain, but they aren't a member of your organization.
+
+Uncaptured users can pose a security threat to your environment since your organization's security settings aren't applied to the user's sessions.
```

Comment:
I had a hard time to understand this part maybe this is just me. Can we please elaborate on this? Possibly:

"Uncaptured users can pose a security threat to your environment since your organization's security settings aren't applied to the user's sessions who are not part of your organization."

Reply:
propose also adding visibility pain point e.g. "...and you will not have visibility into their activity outside your organization"